doc/source/parameters/dem/mesh.rst

```diff
@@ -1,3 +1,30 @@
 Mesh
 -------------------
-The mesh subsection of DEM simulations is identical to the `CFD <https://lethe-cfd.github.io/lethe/parameters/cfd/mesh.html>`_ in Lethe.
\ No newline at end of file
+The mesh subsection of DEM simulations is almost identical to the `CFD <https://lethe-cfd.github.io/lethe/parameters/cfd/mesh.html>`_ in Lethe. There are two additional parameters mainly used in DEM and CFD-DEM simulations. These parameters are ``check diamond cells`` and ``expand particle-wall contact search``.
+
+.. code-block:: text
+
+ subsection mesh
+    set check diamond cells		           = true
+
+    set expand particle-wall contact search = true
+ end
+
+* ``check diamond cells`` parameter enables searching for diamond-shaped boundary cells and adding them to particle-wall contact search cells. The following image shows a diamond-shaped boundary cell colored in red.
+
+.. image:: images/diamond_cell.png
+    :alt: Schematic
+    :align: center
+    :width: 400
+
+
+* ``expand particle-wall contact search`` enables adding the boundary neighbor cells of boundary cells to the particle-wall contact search list. This feature should only be activated in geometries with concave boundaries. (For example, for particles flow inside a cylinder or sphere). The following image shows the boundary neighbor cells (colored in teal) of the read boundary cell.
```

Comment:
Add an explanation as to why this is set to true only for concave geometry (e.g. that otherwise it leads to unphysical contacts with walls that don't exist). You could also add an issue regarding this and maybe we could implement a version that is more intelligent in the future. E.g. it would add the extension only if the normal vector between the walls is at an angle below 90 degrees for example.

Reply:
done. I've added it to my work list in the hackathon.